### PR TITLE
Code style and if structure improvements

### PR DESCRIPTION
**Pull request by regularlabs**
This PR removes some redundant `else` uses (resulting in less indentation and more readability).
And changes stuff like `if (false === isset(...))` to `if (!isset(...))`.

### CODE REVIEW

**Comment by serbanghita** [comment]
```
1) BasicTest::testVersionExtraction with data set #0 ('Mozilla/5.0 (Linux; Android 4...535.19', 'Android', '4.0.4', 4.04)
Failed asserting that false is identical to '4.0.4'.
/home/travis/build/serbanghita/Mobile-Detect/tests/BasicsTest.php:435
```

For some reason tests fail now https://travis-ci.org/serbanghita/Mobile-Detect/jobs/316120843

**Comment by regularlabs** [comment]
Yep, sorry about that.
It failed with a bang. Or better said: failed without the bang (was missing a `!`).
;)

File: Mobile_Detect.php [hunk 1]
```diff
@@ -1317,30 +1318,29 @@ public function version($propertyName, $type = self::VERSION_TYPE_STRING)
...
-                }
+            if (!empty($match[1])) {
+                $version = ($type == self::VERSION_TYPE_FLOAT ? $this->prepareVersionNo($match[1]) : $match[1]);
 
+                return $version;
             }
 
         }
 
-        return false;
```
Comment (shaggybb) [inline Mobile_Detect.php]: In this improvement, you are assuming that the `foreach` will always return something. But what could happen `$match[1]` (line 1336) is always empty? I would keep it as last chance.

Comment (regularlabs) [reply Mobile_Detect.php]: I agree.

**Comment by serbanghita** [comment]
@regularlabs After giving it some thought I decided not to address any code changes at the moment. I want to focus on regex and detection on `2.x`. Thank you for the effort!
@shaggybb thank you for the assistance :+1:

**Pull request closed without merging**